coffeecard/CoffeeCard.WebApi/Logging/Enrichers/UserIdEnricher.cs

```diff
@@ -0,0 +1,40 @@
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace CoffeeCard.WebApi.Logging.Enrichers
+{
+    public class UserIdEnricher : ILogEventEnricher
+    {
+        private const string UserIdPropertyName = "UserId";
+        private readonly IHttpContextAccessor _contextAccessor;
+
+        public UserIdEnricher() : this(new HttpContextAccessor())
+        {
+        }
+
+        private UserIdEnricher(IHttpContextAccessor contextAccessor)
+        {
+            _contextAccessor = contextAccessor;
+        }
```

Comment:
Do you know if we can just use Dependency Injection? I can see you are just following the pattern from `CorrelationIdEnricher` but wondering if below shouldn't be possible:

```suggestion
        public UserIdEnricher(IHttpContextAccessor contextAccessor)
        {
            _contextAccessor = contextAccessor;
        }
```